pages/agent/v3/securing.md

```diff
@@ -149,7 +149,16 @@ exit 1
 
 ## Forcing clean checkouts
 
-By default Buildkite will reuse (after cleaning) a previous checkout. This may not be safe if building commits from untrusted sources (for example, 3rd party pull requests). To force a clean checkout every time, set `BUILDKITE_CLEAN_CHECKOUT=true` in the environment.
+By default Buildkite will reuse (after cleaning) a previous checkout. This may not be safe if building commits from untrusted sources (for example, 3rd party pull requests). To force a clean checkout every time, set `BUILDKITE_CLEAN_CHECKOUT=true` in the environment. Here is an example to force a clean checkout at step level:
+
+```yaml
+steps:
+- label: ":github: Clean Checkout"
+  command: echo "clean checkout"
+  env:
+    BUILDKITE_CLEAN_CHECKOUT: true
+```
+In logs for this step, you will find log group "Cleaning pipeline checkout".
```

Comment:
Style nitpicks:

```suggestion
In the logs for this step, you will find a log group called "Cleaning pipeline checkout."
```

Reply:
Actually, I'm guessing this is meant to be how you confirm the clean happened? If so, perhaps we could say that:

```suggestion
You can check that the clean happened by looking for the "Cleaning pipeline checkout" log group after the step runs.
```